### PR TITLE
Docs: Python Tutorials

### PR DESCRIPTION
**Pull request by ax3l**
List of important python tutorials and prepare section for plugins (will be imported from wiki).

### CODE REVIEW

File: docs/source/python.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+If you are new to python, get your hands on the tutorials of the following important libraries to get started.
+
+- https://www.python.org/about/gettingstarted/
+- https://docs.python.org/3/tutorial/index.html
+
+
+Numpy
+-----
+
+Numpy is the universal swiss army knife for working on ND arrays in Python.
```
Comment (PrometheusPi) [inline docs/source/python.rst]: REQUIRED:
please use lower case `python`

File: docs/source/python.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+-------
+
+Access, share, modify, run and interact with your python scripts from your browser:
+
+https://jupyter.readthedocs.io
+
+
+openPMD-viewer
+--------------
+
```
Comment (PrometheusPi) [inline docs/source/python.rst]: OPTIONAL:
please add a description:
```
A library that automatically reads out our hdf5/adios  file output.
```

File: docs/source/python.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+https://github.com/openPMD/openPMD-viewer/tree/master/tutorials
+
+
+yt-project (dev)
+----------------
+
+Starting with yt 3.4, our HDF5 output, which uses the openPMD markup, can be read with yt.
```
Comment (PrometheusPi) [inline docs/source/python.rst]: OPTIONAL:
read and visualized

File: docs/source/python.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+http://yt-project.org/docs/dev/
+
+
+pyDive (experimental)
+---------------------
+
+pyDive is currently not ready to interpret openPMD directly, but can work on generated raw ADIOS and HDF5 files.
```
Comment (PrometheusPi) [inline docs/source/python.rst]: OPTIONAL:
please describe briefly what `pyDive` does:
```
pyDive allows parallel read and write  access to hdf5 and adios files. It is currently ...
```

File: docs/source/python.rst [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Access, share, modify, run and interact with your python scripts from your browser:
+
+https://jupyter.readthedocs.io
+
+
+openPMD-viewer
+--------------
+
+A library that reads out our hdf5/adios files which a favoured with openPMD markup.
```
Comment (PrometheusPi) [inline docs/source/python.rst]: REQUIRED: 
needs improvement - what do you mean

File: docs/source/python.rst [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+Provides an API to correctly convert units to SI, interpret iteration steps correctly, annotate axis and much more.
+Also provides and interactive GUI for fast exploration via Jupyter notebooks.
```
Comment (PrometheusPi) [inline docs/source/python.rst]: REQUIRED:
`and` -> `an`

File: docs/source/python.rst (hunk 5 above)
Comment (ax3l) [inline docs/source/python.rst]: a double typo

**Pull request merged**